dbc2val/requirements.in

```diff
@@ -0,0 +1,25 @@
+# This file contains top level dependencies pinning a major version.
+# For developing the dbc2val package it makes sense to use this file
+# with 'pip install -r requirements.in' in order to benefit from more
+# recent minor versions of the dependencies.
+#
+# For creating/distributing a binary, all dependencies should
+# be pinned to specific versions in order to provide for a reproducible
+# build.
+# 
+# The 'pip-tools' package's 'pip-compile' command can be used for that
+# purpose. The following command will take the inputs from this file
+# and create a 'requirements.txt' file with pinned versions of all
+# dependencies:
+#
+# pip-compile requirements.in
+#
+
+pyserial ~= 3.5
+cantools ~= 38.0
+pyyaml ~= 6.0
+can-j1939 ~= 2.0
```

Comment:
[can-j1939](https://pypi.org/project/can-j1939/) is not the same as [j1939](https://pypi.org/project/j1939/). Have you tested/investigated that they are compatible?

Reply:
I don't know, but according to the readme can-j1939 https://github.com/juergenH87/python-can-j1939  seems to be a maintained fork of j1939.

License-wise it is fine.

Reply:
I have not done a code introspection but from the README and the code examples, it looks to me as if can-j1939 has been forked off the original j1939. Also, my local tests with a CAN dump file, a custom DBC and mapping file have been successful. The API used by dbc2val also doesn't seem to be broken. So FMPOV this should be a suitable replacement. Do you have any insight pointing in another direction?

Reply:
No insights. I just learned about can-j1939 from your PR. If this is newer, better supported, and something that works for you, by all means bring it in (I don't think anyone else is currently actively working on the j1939 parts, so not much danger of breaking things)

Reply:
Changing to can-j1939 might actually be good, with j1939 there is a dependency problem, see #56. The same exception does not occur if using can-j1939, and no other exception either. I have however not fully tested the j1939 functionally.

Reply:
Looks like there _have_ been some subtle changes between j1939 and can-j1939. I am currently adapting the code accordingly ...

Reply:
Ok, I have pushed some fixes and removed a lot of code which I believe to be obsolete.